metron-deployment/inventory/full-dev-platform/group_vars/all

```diff
@@ -78,7 +79,7 @@ sensor_test_mode: True
 install_pycapa: False
 install_bro: True
 install_snort: True
-install_yaf: True
+install_yaf: False
```

Comment:
Before this change, Snort, Bro and YAF were all installed and started by default. That is my understanding of how it has been for months.  

This PR changes that behavior so that YAF is neither installed nor started.  The comments in your PR only say that you don't want YAF started.  You don't mention anything about YAF not being installed.  

Since there was this gap between what the PR says and what the PR does, I wanted to make sure we bring it to light.

There is a fair amount of documentation that describes to users that when they deploy Metron, they are going to see YAF, Snort and Bro data in the Metron Dashboard.  This PR will change that.  The user will not see YAF data.